api/src/main/java/io/smallrye/stork/api/LoadBalancer.java

```diff
@@ -22,4 +22,10 @@
      *         are deemed invalid for some reason
      */
     ServiceInstance selectServiceInstance(Collection<ServiceInstance> serviceInstances);
+
+    /**
+     *
+     * @return true if the selectServiceInstance method can safely be called in parallel
```

Comment:
```suggestion
     * @return true if the selectServiceInstance method can safely be called concurrently.
```

Reply:
I'll extend this javadoc

Reply:
I change the javadoc entirely, WDYT of the current version?

Reply:
+1 it's clearer now I think.